ci/views.py

```diff
@@ -558,13 +766,6 @@ def identity(request, identity):
                         'Successfully created a subscription.',
                         extra_tags='success'
                     )
-
```

Comment:
I see there are a place where these audit logs are removed in your PR. I think thats a merging problem. They should definitely stay in the code.

Switch to `develop`, do a `git pull`, switch back to your branch and do a `git merge develop`, then fix merge conflicts.